internal/provider/resource_docker_container_funcs.go

```diff
@@ -29,9 +29,9 @@ import (
 )
 
 const (
-	containerReadRefreshTimeout             = 15 * time.Second
-	containerReadRefreshWaitBeforeRefreshes = 100 * time.Millisecond
-	containerReadRefreshDelay               = 100 * time.Millisecond
+	containerReadRefreshTimeoutMillisecondsDefault = 15000
```

Comment:
```suggestion
	containerReadRefreshTimeoutMillisecondsDefault = 15000 * time.Millisecond
```
Maybe we should be consistent here?

Reply:
So I thought about this, but I also use this for the default value set in the provider schema as well (just to avoid having two separate hardcoded integers of the same value) --> and the schema itself wouldn't accept a duration as an input type. Im happy to do whatever you think is best here tho.